server/service/src/sync/translations/invoice_line.rs

```diff
@@ -247,16 +279,28 @@ fn to_legacy_invoice_line_type(_type: &InvoiceLineRowType) -> LegacyTransLineTyp
 #[cfg(test)]
 mod tests {
     use super::*;
-    use repository::{mock::MockDataInserts, test_db::setup_all};
+    use repository::{
+        mock::{mock_outbound_shipment_a, mock_store_b, MockData, MockDataInserts},
+        test_db::setup_all_with_data,
+        KeyValueStoreRow, KeyValueType,
+    };
+    use util::inline_init;
 
     #[actix_rt::test]
     async fn test_invoice_line_translation() {
         use crate::sync::test::test_data::invoice_line as test_data;
         let translator = InvoiceLineTranslation {};
 
-        let (_, connection, _, _) = setup_all(
+        let (_, connection, _, _) = setup_all_with_data(
             "test_invoice_line_translation",
-            MockDataInserts::none().units().items(),
+            MockDataInserts::none().units().items().names().stores(),
```

Comment:
BTW: I've been playing around with adding dependancies to the MockDataInserts in HSH.
So we can just do something like `MockDataInserts::none().stores()` and it will automatically set the dependant data bools to true e.g. units, items, names in this case.
https://github.com/openmsupply/health-supply-hub/blob/3bfc277c063008131aa133b9c3bfd685076f1935/backend/repository/src/mock/mod.rs#L90

Reply:
🙏 I agree this should be automatic, added to cary over. This bit of code has gone through quite a lot of iterations, it's in a bit of a limbo now, i doubt we'll have the time to do any work on it anytime soon, but would be great to tidy it up.